File: 1047/1047 - Tempo de Jogo com Minutos.py
hi,mi,hf,mf = map(int, input().split())
th = hf - hi
tm = mf - mi
if tm < 0: tm += 60; th -= 1
if tm == 0 and th <= 0: th += 24
if th < 0: th += 24
print('O JOGO DUROU {} HORA(S) E {} MINUTO(S)'.format(th,tm))

File: 1047/1047 - Tempo de Jogo com Minutos.lua
hi,mi,hf,mf = io.read("*n", "*n", "*n" ,"*n")

th = hf - hi
tm = mf - mi
if tm < 0 then
    tm = tm + 60
    th = th - 1
end
if tm == 0 and th <= 0 then
    th = th + 24
end
if th < 0 then
    th = th + 24
end
print(string.format('O JOGO DUROU %d HORA(S) E %d MINUTO(S)', th, tm))